analysis/queries.py

```diff
@@ -34,3 +34,19 @@ def get_experiment_data(experiment_names):
         .filter(Trial.preempted.is_(False))
 
     return pd.read_sql_query(snapshots_query.statement, db_utils.engine)
+
+
+def add_nonprivate_experiments_for_merge_with_clobber(experiment_names):
+    """Returns a new list containing experiment names preeceeded by a list of
+    nonprivate experiments in the order in which they were run. This is useful
+    if you want to combine reports from |experiment_names| and all nonprivate
+    experiments."""
+    nonprivate_experiments = db_utils.query(
+        Experiment.name).filter(~Experiment.private).filter(
+            ~Experiment.name.in_(experiment_names)).order_by(
+                Experiment.time_created)
+    nonprivate_experiment_names = [
+        result[0] for result in nonprivate_experiments
```

Comment:
nit: you might also use `result for result, in ...`

Reply:
I didn't know about that syntax thanks!
But I think I'll keep it the way it is, I think it's clearer in some ways (though your suggestion is more explicit about there being one item in the tuple).